purchases/src/test/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsTrackerTest.kt

```diff
@@ -38,13 +48,29 @@ class DiagnosticsTrackerTest {
 
     private lateinit var diagnosticsTracker: DiagnosticsTracker
 
+    private lateinit var context: Context
+    private lateinit var appConfig: AppConfig
+
     @Before
     fun setup() {
+        mockkStatic("com.revenuecat.purchases.common.UtilsKt")
```

Comment:
FFTI. An alternative to this could be to mock the `context.packageManager.getPackageInfo` function (we would need to have a mock for the package manager as well). Personally I prefer to avoid mocking extension functions if possible due to the possibility of forgetting to clear the mocks as you said.

Reply:
Will try to address this in a seppare PR 👍